Document/0x05d-Testing-Data-Storage.md

```diff
@@ -1069,22 +1069,35 @@ setContentView(R.layout.activity_main);
 
 #### Overview
 
-Analyzing the memory can help to identify the root cause of different problems, like for example why an application is crashing, but can also be used to identify sensitive data. This section describes how to check for sensitive data and disclosure of data in general within the process memory.
+Analyzing memory can help developers to identify root causes of different problems, like for example application crashes, etc. However, it can also be used to gain access to sensitive data. This section describes how to check for disclosure of data within the process' memory.
+
+First, you need to identify which sensitive information is stored in memory. Basically, if you have a sensitive asset it's very likely that at some point it is loaded in memory. The objective is to verify that these info has as short exposure as possible. 
```

Comment:
> these info

I think this should be "this information".

> has as short exposure as possible.

I think some word is missing here. Maybe it's better to make it "is exposed as briefly as possible."